envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -124,6 +124,11 @@ class EgressProperties {
     var clusterNotFoundStatusCode = 503
     var handleInternalRedirect = false
     var http2 = Http2Properties()
+    var commonHttp = CommonHttpProperties()
+}
+
+class CommonHttpProperties {
+    var idleTimeout: Duration = Duration.ofSeconds(120)
```

Comment:
Is this default ok? @pbetkier

Reply:
I dunno, where did you get this number from? Perhaps look at the defaults in common http clients?

Reply:
Play framework uses 75s, ibm something 60, some product from f5 - 300, i saw 860 in envoy somewhere so I took a weird average and called it 120 :D

Reply:
I'm convinced ;)